components/PictureList.js

```diff
@@ -0,0 +1,45 @@
+import { Fragment } from "react";
+import Image from "next/image.js";
+import { styled } from "styled-components";
+import pictures from "../db.js";
+
+export default function PictureList() {
+  return (
+    <>
+      <StyledArticle>
+        {pictures.map((picture) => (
+          <ul key={picture.id}>
+            <li>
+              <StyledImage
+                src={picture.image}
+                alt={picture.theme}
+                height={300}
+                width={200}
+              />
+            </li>
+            <StyledTheme>{picture.theme}</StyledTheme>
+          </ul>
```

Comment:
you normally don't want to create a `ul` for every element in your array, but rather wrap the mapping of the array in a `ul` tag. In that way you generate one `li` element per item in the array, and all of them go into one `ul`. You need to add the `key` prop to your li element then.